tutor/src/screens/student-dashboard/event-row.jsx

```diff
@@ -143,7 +143,9 @@ class EventRow extends React.Component {
     const { event, course } = this.props;
     return (
       <>
-        <MobileCell>
+        <MobileCell
+          href={Router.makePathname('viewTask', { courseId: course.id, id: event.id })}
+          onClick={this.onClick}>
           <div className="assignment-title-icon">
```

Comment:
we need to make `MobileCell` be a `a` tag for accessibility   I see that `MobileCell` is only being used here, so we can rename it to `MobileRow` and have it be a styled.a tag